### PR TITLE
#2440 Quickstart VM - prepare service and data

### PR DESCRIPTION
**Pull request by eechih**
### Checklist
- [x] [Review Contribution Guidelines](https://github.com/oharastream/ohara/blob/master/CONTRIBUTING.md)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#authors))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #2440 )
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: vms/quickstart/http/preseed.cfg [hunk 1]
```diff
@@ -40,4 +40,4 @@ d-i passwd/user-password-again password oharastream
 d-i user-setup/allow-password-weak boolean true
 d-i user-setup/encrypt-home boolean false
 d-i passwd/user-default-groups ohara sudo
-d-i passwd/user-uid string 900
\ No newline at end of file
+d-i passwd/user-uid string 1000
```
Comment (chia7712) [inline vms/quickstart/http/preseed.cfg]: nice

File: vms/quickstart/packer-quickstart.json [hunk 2]
```diff
@@ -58,7 +58,7 @@
       "vm_name": "ohara-quickstart-{{user `OHARA_VER`}}",
       "vboxmanage": [
         ["modifyvm", "{{.Name}}", "--cpus", "2"],
-        ["modifyvm", "{{.Name}}", "--memory", "4096"],
+        ["modifyvm", "{{.Name}}", "--memory", "8192"],
```
Comment (chia7712) [inline vms/quickstart/packer-quickstart.json]: :)

**Comment by eechih** [comment]
@vitojeng Addressed your comment, please review again, thanks. The document will be updated in another issue.

**Comment by vitojeng** [comment]
> @vitojeng Addressed your comment, please review again, thanks. The document will be updated in another issue.

+1, thanks for the great pr!

**Pull request merged**